google-disk-expand.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Resize root filesystem if needed
+Before=basic.target
+After=local-fs.target sysinit.target
+DefaultDependencies=no
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/google_disk_expand
```

Comment:
I am wondering if we should rename the service and the script to google_filesystem_expand to be correctly descriptive.

Reply:
thought a lot about naming - decided on this based on being as obvious as possible. the initrd module is expand_root and the service and script are google_disk_expand to be clear about where they come from.

Reply:
SGTM